{
  "name": "test_nested_exception",
  "parameters": [],
  "return_type": null,
  "docstring": null,
  "calls": [
    "io.StringIO",
    "Console",
    "ValueError",
    "exception_text.find",
    "console.file.getvalue",
    "console.print_exception"
  ],
  "variables": [
    "console",
    "value_error_message",
    "exception_text",
    "text_should_contain"
  ],
  "start_line": 138,
  "end_line": 162
}